moped-editor/src/components/GridTable/Filters.js

```diff
@@ -235,25 +231,9 @@ const Filters = ({
 
     const remainingFiltersCount = filtersNewState.length;
 
-    if (remainingFiltersCount === 0) {
-      /* Clear search params since we have no advanced filters */
-      setSearchParams((prevSearchParams) => {
-        prevSearchParams.delete(advancedSearchFilterParamName);
-        prevSearchParams.delete(advancedSearchIsOrParamName);
-
-        return prevSearchParams;
-      });
-    } else if (remainingFiltersCount === 1) {
```

Comment:
we shouldnt be making any changes to the URL search params when a filter is deleted. the URL should only be updated once the Apply button is clicked.